### DISCUSSION TITLE
part-ib/2018-30/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#5] [Question comments]
Part IB 2018 Groups, Rings and Modules: Paper 4, Section I, G

https://questions.tripos.org/part-ib/2018-30/

**Comment by vishalgupta523** [comment]
Paper 4, Section I, Q2G

**Reply by AustenLamacraft** [reply]
One of the weaknesses of the conversion was that the question numbers got stripped off the later papers!
